collectd-config-ansible-role.spec

```diff
@@ -0,0 +1,44 @@
+%global srcname collectd_config_ansible_role
+%global rolename collectd-config
+
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+
+Name:           ansible-%{rolename}
+Version:        0.1
+Release:        1
+Summary:        Ansible role for creating collectd configs
+
+Group:          TODO
+License:        ASL 2.0
+URL:            https://github.com/infrawatch/collectd-config-ansible-role
+Source0:        https://github.com/infrawatch/collectd-config-ansible-role/archive/collectd-config-ansible-role-%{upstream_version}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  git
+
+Requires:       python3dist(ansible)
+
+%description
+
+Ansible role for creating collectd configs
+
+%prep
+%autosetup -n collectd-config-ansible-role -S git
```

Comment:
Is this just future-proofing to ease the addition of patches?

Reply:
Not intentionally, I was following an example [1] for spec files for ansible.

[1]  https://review.rdoproject.org/r/#/c/29389/5/ansible-tripleo-powerflex.spec

Reply:
Yes, it's a common pattern, which all needs fixing.
The benefit of having -S git in autosetup is that it will automatically apply possibly listed patches.

Reply:
Is this a bad pattern? 

having the  -S is good, right? (for future proofing, as Chris pointed out)

Reply:
No, the pattern itself is good. For to reduce the footprint of the build environment, you could replace 
```
BuildRequires: git 
```
with 
```
BuildRequires: git-core
```